src/10-ipfs-0-4-0-released/index.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2016-03-17
+id: 10-ipfs-0-4-0-released
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+- {name: "10-ipfs-0-4-0-released", link: "./" }
+tags: modules
+title: IPFS 0.4.0 has been released
+author: Kyle Drake and Whyrusleeping
+collection: posts
+---
+
+[IPFS 0.4.0](https://github.com/ipfs/go-ipfs/releases/tag/v0.4.0) has been
+released! Among the many changes are a revamped implementation of the IPFS
+communication protocols, increased performance, improvements to IPNS, many
+bugfixes, and a couple new features to make IPFS even more powerful.
+
+### On Breaking Changes
+
+This is a **breaking change** release, with a major refactor and upgrade to the
+IPFS networking protocol. Because of the refactor, **all IPFS daemons earlier
+than 0.4.0 will not be able to communicate with the newest version**. It is
+strongly recommended that everyone running an IPFS node upgrades to the latest
+version as soon as possible, as these nodes will, after a certain time, no
+longer be able to communicate with the majority of the network until they are
+upgraded.
```

Comment:
To avoid giving heart attacks to people, it would be better to link directly from here to "Upgrade docs + a place to get support if needed"